src/work-allocation/release2/components/available-tasks-filter/available-tasks-filter.component.html

```diff
@@ -0,0 +1,18 @@
+<details #filterDetails class="available-tasks-filter govuk-details" data-module="govuk-details" role="group">
```

Comment:
Inside a `release2` directory? 🤦

Reply:
Yeah we need release2 specific components/containers to live in release2 directory. This was agreed before the start of development